parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -436,11 +438,20 @@ private static int getMaxPaddingSize(Configuration conf) {
   }
 
   @Override
-  public OutputCommitter getOutputCommitter(TaskAttemptContext context)
-      throws IOException {
+  public OutputCommitter getOutputCommitter(final TaskAttemptContext context) throws IOException {
     if (committer == null) {
-      Path output = getOutputPath(context);
-      committer = new ParquetOutputCommitter(output, context);
+      final Path output = getOutputPath(context);
+      final String outputCommitterClassName = getConfiguration(context).get(
+        OUTPUT_COMMITTER_CLASS, ParquetOutputCommitter.class.getName());
+      try {
+        committer = (OutputCommitter) Class.forName(outputCommitterClassName)
+          .getDeclaredConstructor(Path.class, TaskAttemptContext.class)
+          .newInstance(output, context);
+      } catch (final ClassNotFoundException | NoSuchMethodException | InstantiationException |
+          IllegalAccessException | InvocationTargetException ex) {
+        throw new BadConfigurationException("could not instantiate output committer class: "
+          + outputCommitterClassName, ex);
+      }
```

Comment:
this can be achieved by extending the ParquetOutputFormat. In what context do you need this?